server.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+const httpServer = require('http-server');
+const open = require('open');
+
+let cache = 3600;
+if (process.env.NODE_ENV === 'production') {
```

Comment:
That's the idea.  At some point, I'll probably modify it to have the same "dev vs dist" flag as `index.jade`.